test/python/tensorflow/README.md

```diff
@@ -4,9 +4,13 @@ tf_unittest_runner is primarily used to run tensorflow python unit tests using n
 
 ## What can be tested
 
- - Python tests using Tensorflow with nGraph embedded
- - Python tests by patching Tensorflow to allow runnings tests using nGraph.
-	```tf_unittest_ngraph.patch```
+ - Python tests using nGraph embedded in Tensorflow (when Tensorflow is built using Option 3 in the link below). 
+ - Python tests using Tensorflow (when built using Option 1 & 2 in the link below) by patching 
+   Tensorflow to allow runnings tests using nGraph. Apply the patch ```tf_unittest_ngraph.patch``` 
+   in your installed tensoflow file:"<local virtualenv>/local/lib/<python>/site-  packages/tensorflow/python/framework/test_util.py"
```

Comment:
the path is showing up as : `/local/lib//site- packages/tensorflow/python/framework/test_util.py`

I think angular brackets mean something in readme, so might need to be escape-coded